src/report.cr

```diff
@@ -0,0 +1,139 @@
+require "uri"
```

Comment:
Could you add a spec for this?

You can use `Process.run` and check for expected output when pointed at the `private_drivers` repository

Reply:
I think it requires the server.
You can use `with_server`, which spins up the server on port 6000. It's bundled with action-controller.
[See further here](https://github.com/spider-gazelle/action-controller/blob/master/spec/curl_context.cr#L37-L61)